EthereumKit/Helper/Network.swift

```diff
@@ -1,6 +1,7 @@
 public enum Network {
     case main
     case ropsten
+    case geth(chainID: Int)
     
```

Comment:
should it be named something like `private` since its private network

Reply:
got it 👍